react/index.js

```diff
@@ -249,7 +249,7 @@ export default class Footer extends Component {
       <footer className={`${footer.footer} bt bw1 b--muted-4 mt4 pv5`}>
         <Container className="justify-center flex">
           <div className="w-100 mw9">
-            <nav className={`${footer.container} pt5-s flex justify-between bg-base c-muted-1`}>
+            <nav className={`${footer.container} ${footer.navegationContainer} pt5-s flex justify-between bg-base c-muted-1`}>
```

Comment:
why add two tokens on the same element?

Reply:
navigation
links
payment

Reply:
because `the footer.container` defined only the space where the information goes. In this footer, the footer.container is used in three different divs (nav, payment and links), and if a dev needs customization only one of then will not make it. Adding other token is enought to enable this.

Reply:
So, footer.container is unnecessary.

Reply:
No, its necessary. Like I said, footer.container defined the space, padding, etc. The footer.navigation can be used to allow the costumization for this specific div.